packages/browser-destinations/src/destinations/upollo/identifyUser/index.ts

```diff
@@ -61,28 +75,17 @@ const identifyUser: BrowserActionDefinition<Settings, UpolloClient, Payload> = {
       defaultObjectUI: 'keyvalue'
     }
   },
-  perform: async (UpClient, { payload, context, settings }) => {
+  perform: async (UpClient, { payload }) => {
     const userInfo = {
       userId: payload.user_id,
       userEmail: payload.email,
       userPhone: payload.phone,
-      userName: payload.name,
+      userName: payload.name ? payload.name : (payload.firstName + ' ' + payload.lastName).trim(),
```

Comment:
A difference here is that we will be sending an empty string, where previously it would have been `undefined`. Im not sure if thats important, But i'd favour keeping `undefined`.

```suggestion
      userName: payload.name ? payload.name : (payload.firstName || payload.lasteName) ? (payload.firstName + ' ' + payload.lastName).trim(): undefined,
```